misc/toml-parser/src/main/java/io/ballerina/toml/api/Toml.java

```diff
@@ -169,19 +206,23 @@ public Toml getTable(String key) {
      */
     public List<Toml> getTables(String key) {
         TopLevelNode tableNode = rootNode.children().get(key);
+        if (tableNode == null || tableNode.kind() != TomlType.TABLE_ARRAY) {
+            return null;
+        }
+        TomlTableArrayNode tomlTableArrayNode = (TomlTableArrayNode) tableNode;
+        List<TomlTableNode> childs = tomlTableArrayNode.children();
         List<Toml> tomlList = new ArrayList<>();
-        if (tableNode.kind() == TomlType.TABLE_ARRAY) {
-            TomlTableArrayNode tomlTableArrayNode = (TomlTableArrayNode) tableNode;
-            List<TomlTableNode> childs = tomlTableArrayNode.children();
-            for (TomlTableNode child : childs) {
-                tomlList.add(new Toml(child));
-            }
-            return tomlList;
+        for (TomlTableNode child : childs) {
+            tomlList.add(new Toml(child));
         }
         return tomlList;
     }
 
-    public List<TomlDiagnostic> getDiagnostics() {
+    public Set<Diagnostic> getDiagnostics() {
```

Comment:
Let's remove the `get` prefix from all methods

Reply:
get prefix should exit for data retrieval methods such as get(), getTable() , getTables(). 
Fixed other methods other than retrieval methods.

Reply:
I see. ack.